active_directory_ldap/komand_active_directory_ldap/actions/modify_groups/action.py

```diff
@@ -24,14 +24,17 @@ def run(self, params={}):
         add_remove = params.get('add_remove')
 
         # Normalize dn
-        dn = formatter.format_dn(dn)[0]
+        dn, search_base = formatter.format_dn(dn)
         dn = formatter.unescape_asterisk(dn)
         self.logger.info(f'Escaped DN {dn}')
         # Normalize group dn
         group_dn = formatter.format_dn(group_dn)[0]
         group_dn = formatter.unescape_asterisk(group_dn)
         self.logger.info(f'Escaped group DN {group_dn}')
 
+        # Check that dn exists in AD
+        self.check_that_user_dn_is_valid(conn, dn, search_base)
```

Comment:
checking that the user dn exists is pointless - just pass the fix=True flag to the ad_add_members_to_groups and it should throw the error - and we're already catching this error.  We're doing this with the remove member option